operator/gitops/argocd/grafana/dashboards/pipeline-service-dashboard.json

```diff
@@ -222,10 +211,6 @@
       "pluginVersion": "9.1.6",
       "targets": [
         {
-          "datasource": {
```

Comment:
I believe this was part of the problem with the Alert panels not rendoring immediately